reference/pickup.v3.yaml

```diff
@@ -0,0 +1,99 @@
+openapi: 3.0.1
+info:
+  title: Pickup
```

Comment:
```suggestion
  title: Pickup Methods
```

Reply:
There may be future plans to expand upon this domain such that it's still under the `pickup` domain but not necessarily relating to pickup methods.